src/main/java/org/amahi/anywhere/adapter/ServerFilesAdapter.java

```diff
@@ -104,8 +109,28 @@ private View newView(ViewGroup container) {
 	private void bindView(ServerFile file, View view) {
 		ImageView fileIconView = (ImageView) view.findViewById(R.id.icon);
 		TextView fileTextView = (TextView) view.findViewById(R.id.text);
+		TextView fileSize = (TextView) view.findViewById(R.id.file_size);
+		TextView fileLastModified = (TextView) view.findViewById(R.id.last_modified);
+               LinearLayout moreInfo = (LinearLayout) view.findViewById(R.id.more_info);
 
 		fileTextView.setText(getFileName(file));
+
+               int size=getFileSize(file);
+
+               if(size==0){
+                 moreInfo.setVisibility(View.GONE);
```

Comment:
folders (directories) have a specific mime type of `text/directory` the size should not be used as an indication of an entry being a folder.